Tools/scripts/build_options.py

```diff
@@ -311,6 +311,7 @@ def __init__(self,
     Feature('Other', 'SDCARD_MISSION', 'AP_SDCARD_STORAGE_ENABLED', 'Enable storing mission on microSD cards', 0, None),
     Feature('Other', 'COMPASS_CAL', 'COMPASS_CAL_ENABLED', 'Enable "tumble" compass calibration', 0, None),
     Feature('Other', 'DRONECAN_SERIAL', 'AP_DRONECAN_SERIAL_ENABLED', 'Enable DroneCAN virtual serial ports', 0, None),
+    Feature('Other', 'OSD_CRSF_LINK_STATS', 'AP_OSD_CRSF_EXTENSIONS_ENABLED', 'Enable OSD panels for CRSF expanded link stats data', 0, "RC_CRSF"),
```

Comment:
should be in OSD section and needs OSD as a dependency in addition to RC_CRSF

Reply:
Done!!